src/contexts/wishlist-context/wishlist-context.js

```diff
@@ -0,0 +1,16 @@
+import {createContext , useContext , useState} from "react"
+
+const WishlistContext = createContext()
+
+const WishlistProvider = ({children}) => {
+  
+    const [wishItems , setWishItems] = useState([])
+    const [wishCount , setWishCount] = useState(0)
```

Comment:
wishCount can be calculated from wishItems.. According to docs, you should not create a separate state for something which can be calculated from existing states